client/js/widgets/seat.js

```diff
@@ -33,6 +33,10 @@ class Seat extends Widget {
       displayedText = displayedText.replaceAll('playerName',this.get('player'))
       setText(this.domElement, displayedText);
     }
+    const scoreboard = widgetFilter(w => w.get('type') == 'scoreboard');
+    for (const board of scoreboard)
+      board.applyDeltaToDOM(null);
```

Comment:
Technically this wouldn't have to be called for _every_ property change but keeping track of the dynamic property names might be overkill.